jmclient/jmclient/blockchaininterface.py

```diff
@@ -665,7 +680,7 @@ def __init__(self, jsonRpc, wallet_name):
         self.shutdown_signal = False
         self.destn_addr = self._rpc("getnewaddress", [])
 
-    def estimate_fee_per_kb(self, N):
+    def estimate_fee_per_kb(self, N: int) -> int:
```

Comment:
nit in 68352d047a636496ec5272c41b4d168ecf481c40:

Same thing about `N` variable name.